rootfs/usr/local/bin/xdebug-enable

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+declare -x XDEBUG_ENABLED
+[[ -z "${XDEBUG_ENABLED}" ]] && XDEBUG_ENABLED="false"
+
+declare -x XDEBUG_REMOTE_ENABLE
+[[ -z "${XDEBUG_REMOTE_ENABLE}" ]] && XDEBUG_REMOTE_ENABLE="1"
+
+declare -x XDEBUG_REMOTE_AUTOSTART
+[[ -z "${XDEBUG_REMOTE_AUTOSTART}" ]] && XDEBUG_REMOTE_AUTOSTART="1"
+
+declare -x XDEBUG_REMOTE_PORT
+[[ -z "${XDEBUG_REMOTE_PORT}" ]] && XDEBUG_REMOTE_PORT="9000"
+
+declare -x XDEBUG_REMOTE_HOST
+[[ -z "${XDEBUG_REMOTE_HOST}" ]] && XDEBUG_REMOTE_HOST="localhost"
+
+declare -x XDEBUG_REMOTE_CONNECT_BACK
+[[ -z "${XDEBUG_REMOTE_CONNECT_BACK}" ]] && XDEBUG_REMOTE_CONNECT_BACK="0"
+
+declare -x XDEBUG_IDEKEY
+[[ -z "${XDEBUG_IDEKEY}" ]] && XDEBUG_IDEKEY="PHPSTORM"
+
+if [[ ${XDEBUG_ENABLED} == "true" || ${XDEBUG_ENABLED} == "1" ]]; then
```

Comment:
Defining default variables within entrypoint is totally valid, but this kind of logic should be part of cmd

Reply:
not sure if you get the intention - this logic is required to put the correct configuration in place before invoking any command